superset-frontend/packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-table/TableStories.tsx

```diff
@@ -86,6 +87,7 @@ function loadData(
       page_length: pageLength,
       show_cell_bars: showCellBars,
       include_search: includeSearch,
+      rearrange_columns: rearrangeColumns,
```

Comment:
Can you change the variable name in this file as well?

Reply:
omg. I grepped the plugin folder for it :laughing: